src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -643,15 +694,37 @@ ApiGlobe.prototype.setCameraTargetGeoPosition = function setCameraTargetGeoPosit
  * The level has to be between the [getMinZoomLevel(), getMaxZoomLevel()].
  * The zoom level and the scale can't be set at the same time.
  * <iframe width="100%" height="400" src="//jsfiddle.net/iTownsIGN/7yk0mpn0/embedded/" allowfullscreen="allowfullscreen" frameborder="0"></iframe>
- * @param {Position} pPosition - The detailed position in the scene.
- * @param      {boolean}  isAnimated  Indicates if animated
- * @return     {Promise}
- */
-ApiGlobe.prototype.setCameraTargetGeoPositionAdvanced = function setCameraTargetGeoPositionAdvanced(pPosition, isAnimated) {
+ * @param {Position} position
+ * @param {Number}  position.longitude  Coordinate longitude WGS84 in degree
+ * @param {Number}  position.latitude  Coordinate latitude WGS84 in degree
+ * @param {Number}  [position.tilt]  Camera tilt in degree
+ * @param {Number}  [position.heading]  Camera heading in degree
+ * @param {Number}  [position.range]  The camera distance to the target center
+ * @param {Number}  [position.level]  level,  ignored if range is set
+ * @param {Number}  [position.scale]  scale,  ignored if the zoom level or range is set. For a scale of 1/500 it is necessary to write 0,002.
+ * @param {boolean}  isAnimated  Indicates if animated
+ * @return {Promise}
+ */
+ApiGlobe.prototype.setCameraTargetGeoPositionAdvanced = function setCameraTargetGeoPositionAdvanced(position, isAnimated) {
```

Comment:
Not totally working for the moment (tested in web dev console) :
- range parameter is strangely handled (value given doesn't fit with the view)
- heading and tilt parameters not taken into account